flutter/lib/benchmark/benchmark.dart

```diff
@@ -135,13 +133,11 @@ class BenchmarkList {
     List<pb.BenchmarkSetting> benchmarkSettings,
   ) {
     for (final task in mlperfConfig.task) {
-      for (final model in task.model) {
-        final benchmarkSetting = benchmarkSettings
-            .singleWhereOrNull((setting) => setting.benchmarkId == model.id);
-        if (benchmarkSetting == null) continue;
+      final benchmarkSetting = benchmarkSettings
+          .singleWhereOrNull((setting) => setting.benchmarkId == task.id);
+      if (benchmarkSetting == null) continue;
```

Comment:
Can you add warning message here? E.g. 
`
No matching benchmark setting for task ${task.id}
`

Reply:
Done.